static/public/images/badges/uk-debt.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 162.999 136.113"><path fill="#E3272A" d="M70.204 124.96V79.935H7.664v-7.21h69.75v52.233"/><path fill="#00578A" d="M10.9 13.817h11.484l53.65 43.287-4.527 5.614"/><path fill="#00578A" d="M7.665 64.305v-7.21h62.54l7.208 7.21M144.778 124.96L87.224 79.133l4.49-5.64 62.642 49.878v1.587"/><path fill="#00578A" d="M85.866 124.96V72.71h68.49v7.208h-61.28v45.04"/><path fill="#E3272A" d="M85.866 64.305V13.817h7.21v43.28h61.28v7.208"/><path fill="#00578A" d="M70.204 57.096v-43.28h7.21v50.49"/><path fill="none" stroke="#E3272A" stroke-width="8" stroke-miterlimit="10" d="M89.04 60.7l21.875-13.938 5.25-20.75 22.5 5.25 19.25-21.5M73.81 76.762l-27.145 17.5-2.197 22-26.303-.25-10.5 18.093"/><path fill="none" stroke="#E3272A" stroke-width="6" stroke-miterlimit="10" d="M160 23.095v-15h-15"/></svg>
```

Comment:
I think its probably best to remove the `xmlns` stuff like this doc suggests: https://github.com/guardian/frontend/blob/master/docs/03-dev-howtos/04-inline-svgs.md